src/pages/success-stories/constitutiondao.page.tsx

```diff
@@ -1,9 +1,110 @@
+import { CaseStudiesHeader } from 'components/CaseStudies/CaseStudiesHeader'
+import { CaseStudyContentContainer } from 'components/CaseStudies/CaseStudyContentContainer'
+import ExternalLink from 'components/ExternalLink'
+import SectionHeader from 'components/SectionHeader'
 import { AppWrapper } from 'components/common'
+import { CASE_STUDY_PROJECTS } from 'constants/successStoryProjects'
+import Image from 'next/image'
 
 export default function ConstitutionDAOPage() {
+  const constitutionSuccessStoryProject = CASE_STUDY_PROJECTS[0]
+
+  const fundingConfig: CaseStudiesConfigurationProps = {
+    cycleNumber: 1,
+    cycles: t`Unlocked`,
+    payouts: t`Unlimited`,
+    tokenName: 'PEOPLE',
+    tokenIssuanceRate: '1,000,000',
+    reservedRate: 0,
+    issuanceReductionRate: 0,
+    redemptionRate: 100,
+    ownerTokenMinting: false,
+    editDeadline: t`No deadline`,
+  }
+
   return (
     <AppWrapper>
-      <h1>Hello</h1>
+      <CaseStudiesHeader
+        header="/assets/constitution-casestudy-banner.png"
+        title="ConstitutionDAO"
+        subtitle={
+          <Trans>
+            How a group of friends on the internet raised over $46 million to
+            buy the U.S. Constitution
+          </Trans>
+        }
+        totalRaised={11613}
+        totalPayments={19121}
+        summary={
+          <Trans>
+            ConstitutionDAO was a single purpose acquisition DAO that raised
+            over $40M in an attempt to buy one of the last thirteen remaining
+            copies of the U.S. Constitution at Sotheby’s on November 18th, 2021.
+            What started as a joke in a group chat became a viral internet
+            sensation. Thousands were onboarded into crypto as a result and the
+            fundraiser gave the world a glimpse of the power of blockchain
+            technology, decentralization, and new forms of governance.
+          </Trans>
+        }
+        projectUrl="/p/constitutiondao"
+        fundingConfig={fundingConfig}
+      />
+      <CaseStudyContentContainer
+        currentProject={constitutionSuccessStoryProject}
+      >
+        <p>
+          <Trans>
+            With an auction estimate of $10-15 million at Sotheby’s,
+            ConstitutionDAO wanted to raise as much as possible and decided to
+            set Payouts to Unlimited. In order to build trust with potential
+            supporters, the campaign set up a 9/13 multisig with core team
+            members and well-known figures in web3. Reserved Rate was also set
+            to 0 to guarantee that everyone had equal access to PEOPLE tokens
+            which would govern the document if they won the auction.
+          </Trans>
+        </p>
+
+        <Image
+          src="/assets/constitution-casestudy-banner2.png"
+          alt="The Gang Buys the Constitution"
+          width={592}
+          height={256}
+          className="pb-5"
+        />
+        <div>
+          <SectionHeader text={t`Success`} />
```

Comment:
Try to use `<Trans>` as much as possible - no point having `t` when we don't need it.

Reply:
Got it, didn't change for this but will keep in mind for future

Reply:
Done where possible